test/CommonLibTests/ConfigUtilityTests.cpp

```diff
@@ -89,4 +89,18 @@ namespace ConfigUtilityTests
         EXPECT_EQ(hr, S_OK);
         EXPECT_STREQ(handlerVersion.QueryStr(), L"value2");
     }
+
+    TEST(ConfigUtilityTestSingle, IgnoresFailedGetElement)
+    {
+        STRU handlerVersion;
+
+        auto element = std::make_unique<NiceMock<MockElement>>();
+        ON_CALL(*element, GetElementByName(_, _))
+            .WillByDefault(DoAll(testing::SetArgPointee<1>(nullptr), testing::Return(HRESULT_FROM_WIN32( ERROR_INVALID_INDEX ))));
+
+        HRESULT hr = ConfigUtility::FindHandlerVersion(element.get(), &handlerVersion);
```

Comment:
Anyway to check the log in this test?

Reply:
There might be by collecting stdout but I don't think it's required here.